src/Support/PHPUnit/EventSubscribers/SwaggerSaveDocumentationSubscriber.php

```diff
@@ -2,25 +2,13 @@
 
 namespace RonasIT\AutoDoc\Support\PHPUnit\EventSubscribers;
 
-use Illuminate\Contracts\Console\Kernel;
-use PHPUnit\Event\Application\Finished;
-use PHPUnit\Event\Application\FinishedSubscriber;
-use RonasIT\AutoDoc\Services\SwaggerService;
+use PHPUnit\Event\TestRunner\ExecutionFinished;
+use PHPUnit\Event\TestRunner\ExecutionFinishedSubscriber;
 
-final class SwaggerSaveDocumentationSubscriber implements FinishedSubscriber
+final class SwaggerSaveDocumentationSubscriber implements ExecutionFinishedSubscriber
 {
-    public function notify(Finished $event): void
+    public function notify(ExecutionFinished $event): void
     {
-        $this->createApplication();
-
-        app(SwaggerService::class)->saveProductionData();
-    }
-
-    protected function createApplication(): void
-    {
-        $app = require base_path('bootstrap/app.php');
-
-        $app->loadEnvironmentFrom('.env.testing');
-        $app->make(Kernel::class)->bootstrap();
+        shell_exec('php artisan swagger:push-documentation');
```

Comment:
the console command is just a way to save the new documentation content using driver. The core logic of documentation saving is provided by the SwaggerService.

The subscriber class is another way to trigger the documentation saving, no need to use console command here.